client/my-sites/upgrades/cart/cart-item.jsx

```diff
@@ -136,6 +136,8 @@ const CartItem = React.createClass( {
 		if ( this.props.cartItem.bill_period && this.props.cartItem.bill_period !== -1 ) {
 			if ( isMonthly( this.props.cartItem ) ) {
 				name += ' - ' + this.props.translate( 'monthly subscription' );
+			} else if ( isTheme( this.props.cartItem ) ) {
+				name += ' - ' + this.props.translate( 'never expires' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **36** times:
`translate( 'Never expires' )` *ES Score: 19*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/pull/15523)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).